src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/BaseQualityRankSumTest.java

```diff
@@ -25,6 +26,11 @@
  *
  */
 public final class BaseQualityRankSumTest extends RankSumTest {
+
+    public BaseQualityRankSumTest(final AnnotationGroup... groups) {
+        super(groups);
```

Comment:
It strikes me as cumbersome and awkward to be passing the `AnnotationGroup` (which is just `Standard` or `ActiveRegionBased` in this branch) in to the constructor for each and every annotation class. Why not just use a marker interface like in GATK3 (eg., `Foo implements ActiveRegionBasedAnnotation`), plus `instanceof` checks in the VA engine?